### PR TITLE
Readme - update publishing details

### LINKED ISSUE #109: Update Readme package publishing instructions
**Describe the bug**
Currently the Readme's Publishing Packages suggests we should publish all of the packages at the same time: https://github.com/BBC-News/psammead#publishing-packages

We should update it to state: 

**Set up your account**

- Create an npm account with your bbc email address. 
- In your profile settings, set up Two Factor Authentication. Enable it for authorization and publishing. 
- Back up your recovery codes for your account
- Join the BBC npm org by following the process here: https://github.com/bbc/npm When you're added to the org and signed in, you should be able to see all public and private `@bbc` packages here: https://www.npmjs.com/settings/bbc/packages

**Publishing**

- In your terminal, run `npm login` and follow the steps for authentication
- `npm publish packages/<PATH_TO_PACKAGE>`
e.g. `npm publish packages/components/psammead-paragraph`

**Making package public**

- `npm access public @bbc/<PACKAGE_NAME> --otp=<YOUR_2FA_CODE>`

- [x] Initially labelled with ["bug"](https://github.com/BBC-News/psammead/labels/bug)

### PR DESCRIPTION
**Pull request by sareh**
Resolves #109

Updates package publishing information in the Readme so that it is accurate.

- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,14 +30,19 @@ npm run build
...
+- Join the BBC npm org by following the process here: https://github.com/bbc/npm When you're added to the org and signed in, you should be able to see all public and private `@bbc` packages here: https://www.npmjs.com/settings/bbc/packages
 
-### Without 2FA
+#### Publishing a package
 
-```
-npm run publish
-```
+- In your terminal, run `npm login` and follow the steps for authentication
+- `npm publish packages/<PATH_TO_PACKAGE>`
```
Comment (bcmn) [inline README.md]: Repeating my comment on the issue:

> >npm publish packages/<PATH_TO_PACKAGE>
> >e.g. npm publish packages/components/psammead-paragraph
>
> I think this line should also include the --otp=<YOUR_2FA_CODE>, as I got an error & re-prompted for it. 😄

Comment (bcmn) [reply README.md]:
```suggestion
- `npm publish packages/<PATH_TO_PACKAGE> --otp=<YOUR_2FA_CODE>`
```

Comment (sareh) [reply README.md]: Thanks!

**Review by bcmn — APPROVED** [review]

**Review by pjlee11 — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM

**Pull request merged**